README.md

```diff
@@ -1 +1,72 @@
 # PISP Demo Server (Work in Progress)
+
+A server used to communicate with the Mojaloop network to facilitate account linking and transactions for the PISP demo app (see: https://github.com/mojaloop/pisp-demo-app-flutter).
+
+# Quick Setup Guide
+
+1. Firebase SDK Admin Key
+
+Follow the instructions outlined here: https://firebase.google.com/docs/admin/setup to obtain a .json file containing your secret key.
+Rename it to be `serviceAccountKey.json` and put it in `{project-directory}/secret/` so that the path to the file is: `{project-directory}/secret/serviceAccountKey`.
```

Comment:
Actually for this one, it could be customised from the config. Perhaps can mention that people can just put the path in the env variable or config file, but the default one is `{project-directory}/secret/serviceAccountKey`?